### PR TITLE
Added collection_by_id to GoogleDrive::Session

### PR DESCRIPTION
**Pull request by sam2themax**
Added `collection_by_id` to GoogleDrive::Session because `collection_by_url` doesn't seem to work with custom domain gsuite collections.

### CODE REVIEW

File: lib/google_drive/session.rb [hunk 1]
```diff
@@ -418,6 +418,24 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
+    # Returns GoogleDrive::Collection with given +key+.
+    #
+    # e.g.
+    #   # https://drive.google.com/drive/u/2/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J
+    #   session.collection_by_key("1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J")
+    def collection_by_key(key)
```
Comment (gimite) [inline lib/google_drive/session.rb]: Can you use `id` instead of `key` e.g., collection_by_id? `key` is a legacy terminology from the old Spreadsheet API. So it shouldn't be used for things other than spreadsheets. Do not add `key` method to GoogleDrive::File for the same reason.

File: lib/google_drive/session.rb [hunk 2]
```diff
@@ -418,6 +418,24 @@ def collection_by_url(url)
 
     alias folder_by_url collection_by_url
 
+    # Returns GoogleDrive::Collection with given +key+.
+    #
+    # e.g.
+    #   # https://drive.google.com/drive/u/2/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J
```
Comment (gimite) [inline lib/google_drive/session.rb]: Can you use more normal URL here as an example, not one with multi account sign in? e.g.:
https://drive.google.com/drive/folders/1rPPuzAew4tO3ORc88Vz1JscPCcnrX7-J

File: test/test_google_drive.rb [hunk 3]
```diff
@@ -394,7 +397,7 @@ def get_session
         )
       end
 
-      @@session = GoogleDrive::Session.from_config(config_path)
+      @@session = GoogleDrive::Session.from_config('config.json')
```
Comment (gimite) [inline test/test_google_drive.rb]: Why did you change here?

File: lib/google_drive/session.rb (hunk 1 above)
Comment (sam2themax) [reply lib/google_drive/session.rb]: Ah, thanks for the heads up.  I haven't worked with the Google API much before this, so I was just following the pattern I saw in the spreadsheets file.  I'll change it.

File: test/test_google_drive.rb (hunk 3 above)
Comment (gimite) [reply test/test_google_drive.rb]: The error message indicates that **test**/config.json is invalid as JSON. Maybe you put a broken file there? Note that you must put the config file inside "test" directory.

Comment (sam2themax) [reply test/test_google_drive.rb]: Ah whoops...missed the part about the "test" directory.  Thanks for the help -- I've restored the `config_path` variable now.

**Comment by gimite** [comment]
Thanks! Merged.
